api/specs/webserver/openapi.yaml

```diff
@@ -253,6 +257,13 @@ paths:
   /tags/{tag_id}:
     $ref: "./openapi-tags.yaml#/paths/~1tags~1{tag_id}"
 
+  # FOLDERS -------------------------------------------------------------------------
+  /folders:
```

Comment:
This is more in the direction that i think it should go. But think instead of a file system and not just folders as resources. (see below)

Reply:
I found this https://docs.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/path ... for inspiration

Reply:
As a plus ... having an abstract file-system will allow us as well (if we like) reuse it to any other type of resources like files etc as well